pages/resources/channel.mdx

```diff
@@ -109,6 +109,7 @@ Type `10`, `11` and `12` are only available in API v9 and above.
 | 1 << 14     | BROADCASTING                               | This group DM is used for broadcasting a live stream                                                                                                            |
 | 1 << 15     | HIDE_MEDIA_DOWNLOAD_OPTIONS                | This media channel has the embedded download options hidden for media attachments                                                                               |
 | 1 << 16     | JOIN_REQUEST_INTERVIEW_CHANNEL             | This group DM is used for [guild join request interviews](https://support.discord.com/hc/en-us/articles/23187611406999-Guilds-FAQ#h_01HXW2MCD0BT70FB9SRV1YRPBV) |
+| 1 << 17     | OBFUSCATED                                 | This channel's name is obfuscated because user doesn't have access to it                                                                                        |
```

Comment:
```suggestion
| 1 << 17     | OBFUSCATED                                 | This channel is unavailable for the current user                                                                                        |
```

Can you add a footnote saying something like ``The channel's name is always returned as `___hidden___`.``